x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/columns.tsx

```diff
@@ -6,195 +6,135 @@
  */
```

Comment:
should methods transformed to hooks be renamed and moved to hooks area of application? (I've noticed all hooks were placed before in `x-pack/plugins/security_solution/public/detections/containers/detection_engine`

Reply:
I don't mind renaming and moving those hooks, but considering I'm not the only one currently working on the rules table, that could make merge conflict resolving much harder.

Reply:
Do we then need to keep separately components/hooks/utils etc, in designed folders as we mostly do know? Or it's not mandatory?

Either way fine by me, just would be good to agreed on one approach to follow.

> but considering I'm not the only one currently working on the rules table, that could make merge conflict resolving much harder.

Looks like it won't be easy anyway, there are a lot of changes.
I can move it after my merge or keep there, depends on what we would agree on structure

Reply:
> Do we then need to keep separately components/hooks/utils etc, in designed folders as we mostly do know? Or it's not mandatory?

Let's chat about that at Tech Time. @vitaliidm would you mind adding it to the agenda?

Reply:
as discussed on a call: let's structure our code by features. I.e. put all related components/hooks/utils into feature named folder.